int32 finger_type
geometry_msgs/Point[] joints
uint32 is_extended
